tests/test_ufrm.py

```diff
@@ -93,6 +99,28 @@ def test_ufrm_regression_no_infrastructure(self):
         expected_result = 156070.36
         self.assertAlmostEqual(result_sum, expected_result, places=0)
 
+        result_vector = gdal.OpenEx(os.path.join(
+            args['workspace_dir'], 'flood_risk_service_Test1.shp'),
+            gdal.OF_VECTOR)
+        result_layer = result_vector.GetLayer()
+        result_feature = next(result_layer)
+
+        # Check that only the two expected fields are there.
+        self.assertEqual(
+            set(('rnf_rt_idx', 'rnf_rt_m3')),
+            set(field.GetName() for field in result_layer.schema))
+
+        result_val = result_feature.GetField('rnf_rt_m3')

```

Comment:
Is it worth checking the other field's value also?

Reply:
👍  91bbc64d